ui/app/AppLayouts/Onboarding/views/KeysMainView.qml

```diff
@@ -20,12 +23,13 @@ Item {
     id: root
 
     property StartupStore startupStore
+    property MetricsStore metricsStore
 
     Component.onCompleted: {
         if (button1.visible) {
             button1.forceActiveFocus()
         }
-        enableCentricMetrics.checked = root.startupStore.isCentralizedMetricsEnabled()
+        metricsEnablePopup.visible = true
```

Comment:
I think we only want to show this popup on the first run, but it's worth checking with @xAlisher

Reply:
It is shown only during creating new account.